app/controllers/user.controller.js

```diff
@@ -1,23 +1,17 @@
-const { createUser } = require('../service/user');
 const db = require('../models');
 
 const { User } = db;
 
-// Update a user by the id in the request
+// Update a user
 exports.updateOne = (req, res, next) => {
-  const {
-    params: { id },
-  } = req;
+  const { user } = req;
 
-  User.findById(id)
-    .then((user) => {
-      user.set(req.body);
-      return user.save();
-    })
-    .then((user) => {
+  user
+    .update(req.body)
+    .then((updatedUser) => {
```

Comment:
Tbh, this one I dont understand. Where are u connecting to the db? Dont you need User.findById(id)?